src/agent/TagAgent.tsx

```diff
@@ -27,17 +35,16 @@ export async function getSuggestionTag(title: string) {
     Input Title:
     "{title}"
 
-    Output Format (JSON):
-    {{
-      "tags": ["tag1", "tag2", "tag3", "tag4", "tag5"]
-    }}
-    `,
+    {format_instructions}
+    `.trim(),
   );
-  const outputParser = new JsonOutputParser();
 
-  const chain = prompt.pipe(model).pipe(outputParser);
+  const chain = RunnableSequence.from([prompt, model, parser]);
 
-  const response = await chain.invoke({ title });
+  const response = await chain.invoke({
+    title,
+    format_instructions: parser.getFormatInstructions(),
+  });
 
```

Comment:
```suggestion
const response = await chain.invoke({ title });
```

이런식으로 기존 방식으로 가면 될 것 같네용..?!